packages/message-system/src/config/config.v1.json

```diff
@@ -1,8 +1,28 @@
 {
     "version": 1,
-    "timestamp": "2022-11-28T00:00:00+00:00",
-    "sequence": 17,
+    "timestamp": "2022-11-14T00:00:00+00:00",
+    "sequence": 24,
     "actions": [
+        {
+            "conditions": [],
+            "message": {
+                "id": "c3196202-b796-4c57-abdb-ac68c15ae341",
+                "priority": 99,
+                "dismissible": true,
+                "variant": "critical",
+                "category": ["feature", "context"],
+                "content": {
+                    "en-GB": "CoinJoin disabled for a good reason!",
+                    "en": "CoinJoin disabled for a good reason!",
+                    "es": "CoinJoin disabled for a good reason!",
+                    "cs": "CoinJoin disabled for a good reason!",
+                    "ru": "CoinJoin disabled for a good reason!",
+                    "ja": "CoinJoin disabled for a good reason!"
+                },
+                "feature": [{ "domain": "coinjoin", "flag": false }],
```

Comment:
add something like forceAllow param? to allow feature flag cancelable by user

Reply:
let's implement it together with the feature flag that should be cancelable - CoinJoin doesn't need it